GameData/RationalResourcesParts/CRP/SMX_DrillOMatics.cfg

```diff
@@ -91,4 +91,20 @@
 		@StopActionName = Stop Water Harvester
 		@ToggleActionName = Toggle Water Harvester
 	}
-}
\ No newline at end of file
+}
```

Comment:
This change is not necessary. Your pumps are already handled at the bottom of [SMX_Intake.cfg](https://github.com/JadeOfMaar/RationalResources/blob/master/GameData/RationalResourcesParts/CRP/SMX_Intake.cfg#L160).